From 79b7a58a989a3afd3fe827aa044e71a0792ceec1 Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Tue, 7 Jun 2022 17:41:40 +0800
Subject: [PATCH] sw64: remove redundant Kconfig source
Patch-mainline: NO, OTHERS
References: openEuler-22.03-LTS
Modified-by-SEL: No


Sunway inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Remove redundant Kconfig source to avoid compiler warning.

Signed-off-by: Mao Minkai <maominkai@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/Kconfig | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/sw_64/Kconfig b/arch/sw_64/Kconfig
index a81d388ed304..318a423d87dd 100644
--- a/arch/sw_64/Kconfig
+++ b/arch/sw_64/Kconfig
@@ -419,9 +419,6 @@ config ARCH_DISCONTIGMEM_ENABLE
 	  or have huge holes in the physical address space for other reasons.
 	  See <file:Documentation/vm/numa> for more.
 
-source "kernel/Kconfig.preempt"
-
-
 config NUMA
 	bool "NUMA Support"
 	depends on SMP && !FLATMEM
-- 
2.34.1

